ert_shared/plugins/hook_specifications/logging.py

```diff
@@ -2,11 +2,9 @@
 
 
 @hook_specification
-def add_log_handle(logging):
+def add_log_handle_to_root():
     """
-    Modify ert's logging instance, for example adding handlers and changing formatting.
+    Adding handlers to the root logger in ert.
 
-    :param logging: A handle to the ert logging instance.
-    :type logging: :module:`logging`
-    :return: None
+    :return: logging.Handler
```

Comment:
Should this be a list of logging.Handler(s) instead?

Reply:
No, pluggy will automatically add return values to a list as there could be several plugins serving up different handlers.